QuantLib/ql/termstructures/volatility/optionlet/optionletstripper.hpp

```diff
@@ -40,6 +40,7 @@ namespace QuantLib {
       public:
         //! \name StrippedOptionletBase interface
         //@{
+        enum Model{Normal, ShiftedLognormal};
```

Comment:
Ok, I see. I had just looked and the signature and thought the displacement was used... I'd remove it from the signature of the engine. It might be confusing.

As for the `if/else`: I think is ok. I'd drop the "shifted" from the `enum` and just keep `Normal` and `Lognormal`, though. I don't think it makes much sense to have `Normal` and `ShiftedNormal` in case we start supporting a displacement. It's `Normal` anyway.